romancal/dq_init/dq_init_step.py

```diff
@@ -83,6 +83,21 @@ def process(self, input):
                 **reference_file_models,
             )
 
+            # copy original border reference file arrays (data and dq)
+            # to their own attributes. they will also remain attached to
+            # the science data until they are trimmed at ramp_fit
+            # these arrays include the overlap regions in the corners
+
+            output_model.border_ref_pix_right = output_model.data[:, -4:]
+            output_model.border_ref_pix_left = output_model.data[:, :4]
+            output_model.border_ref_pix_top = output_model.data[:4, :]
+            output_model.border_ref_pix_bottom = output_model.data[-4:, :]
+
+            output_model.dq_border_ref_pix_right = output_model.pixeldq[:, -4:]
+            output_model.dq_border_ref_pix_left = output_model.pixeldq[:, :4]
+            output_model.dq_border_ref_pix_top = output_model.pixeldq[:4, :]
+            output_model.dq_border_ref_pix_bottom = output_model.pixeldq[-4:, :]
+
```

Comment:
These 8 lines need tests.

Reply:
Correct the test_ramp_fit.py needs to be expanded to test the new code.

Reply:
At some level these outputs are validates by the schemas., not the size but the dimension.

Reply:
I suggest we update the ramp_fit tests in a different PR and "unxfail"  them.

Reply:
I think it is fine to XFail these and put tests in another PR. I do think we need tests beyond validation, as it is easy to accidentally grab the wrong values (or mess up their order), which the tests should cover.